### DISCUSSION TITLE
Lesson 4: Could not find files for the given pattern(s). <class 'web3._utils.datatypes.Contract'>

### DISCUSSION THREAD

**Discussion by Panosfunk** [smartcontractkit/full-blockchain-solidity-course-py#1574] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/36935711/178159635-5be099cd-895f-4eaa-ac91-2c58193107aa.png)
This is my code.

And this is the error im getting. 

python .\deploy.py
INFO: Could not find files for the given pattern(s).
<class 'web3._utils.datatypes.Contract'>

Can anyone help? I'd happily give more context, just ask.

**Comment by cromewar** [comment, marked as answer]
Hello @Panosfunk 
`INFO: Could not find files for the given pattern(s).` is not an actual error, just a info about the translation between instructions on cmd and powershell.
But you don't have anything to worry about ;)
